TravelBug/src/WebPages/loginPage/Loginpage.css

```diff
@@ -0,0 +1,64 @@
+.login-container {
+  font-family: roboto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.login-inputs {
+  background-color: var(--login-container-color);
+  width: 25rem;
+  height: 35rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-shadow: var(--shadow-box);
+  align-items: center;
+}
+
+.primary-button {
+  border: none;
+}
+
+.primary-button:hover {
+  background-color: var(--login-hover);
+}
+a {
+  color: rgb(255, 255, 255);
```

Comment:
you can create variable for these two rgb values for consistency.

Reply:
done